tensorboard/program_test.py

```diff
@@ -62,6 +62,18 @@ def testMakeServerBlankHost(self):
         self._StubApplication(),
         self.make_flags(host='', port=0, path_prefix=''))
     self.assertStartsWith(server.get_url(), 'http://')
+    
+    def testPathPrefixSlash(self):
+    #Test that checks the path prefix ends with one trailing slash
+    server = program.WerkzeugServer(
+        self._StubApplication(),
+        self.make_flags(host='127.0.0.1', port=0, path_prefix='/test'))
+    self.assertStartsWith(server.get_url(), 'http://127.0.0.1:0/test/')
```

Comment:
Rather than assertStartsWith, we really need an `assertEndsWith` so that we can check the exact path and make sure there are no extra slashes.  And that way we can avoid asserting on the exact host and port, which aren't relevant to this test.

Can you implement an `assertEndsWith` helper in this file as a method of the TestCase class?  It should look exactly like `assertStartsWith` except that you can replace "start" with "end" everywhere:

https://github.com/tensorflow/tensorboard/blob/2aa2ba70eec793442438d66574304f64b990127b/tensorboard/test.py#L80-L85

You may need to merge the master branch into your pull request branch in order to have the `test.py` file.

Reply:
@nfelt  i updated the assertStartsWith to assertEndsWith in my code, but i lost you at merging the master branch into your pull request branch, can you help me with this??

Reply:
@nfelt i did everything else just cant get the `test.py` in my pull request branch. Please help me this!

Reply:
Here's some instructions: https://help.github.com/en/articles/merging-an-upstream-repository-into-your-fork

Basically you should:
1. check out your PR branch - https://github.com/amanp592/tensorboard/tree/patch-1 - so `git checkout patch-1` locally
2. run `git pull https://github.com/tensorflow/tensorboard.git master` and accept the merge
3. make changes to `test.py` as needed, which should now exist
4. `git push` your updated branch patch-1